### PR TITLE
Ship 0.13.2

### PR DESCRIPTION
**Pull request by zpao**
:boom: Don't merge this, I'll fast-forward it once everything is done :boom: 

- [x] Cherry-pick all of the things from [the list](https://github.com/facebook/react/issues?utf8=%E2%9C%93&q=milestone%3A0.13.2)
- [x] Bump version
- [x] Update changelog
- [x] Update readme links
- [x] blog post
- [ ] actually ship it

### CODE REVIEW

**Comment by spicyj** [comment]
Let's get #3675 in too.

**Comment by zpao** [comment]
Changelog seem ok @spicyj?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,33 @@
...
+* Added `strokeDashoffset`, `flexPositive`, `flexNegative` to the list of unitless CSS properties
+* Added support for more DOM properties:
+  * `scoped` - for `<style>` elements
+  * `high`, `low`, `optimum` - for `<meter>` elements
+  * `unselectable` - IE specific property to prevent user selection
+
+#### Bug Fixes
+
+* Fixed a case where re-rendering after rendering null didn't properly pass context
+* Fixed a case where re-rendering after rendering null didn't properly update style
```
Comment (spicyj) [inline CHANGELOG.md]: This was if you did `style={null}`, right? Not if you rendered null.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,33 @@
...
+
+### React Core
+
+#### New Features
+
+* Added `strokeDashoffset`, `flexPositive`, `flexNegative` to the list of unitless CSS properties
+* Added support for more DOM properties:
+  * `scoped` - for `<style>` elements
+  * `high`, `low`, `optimum` - for `<meter>` elements
+  * `unselectable` - IE specific property to prevent user selection
```
Comment (spicyj) [inline CHANGELOG.md]: IE-specific

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,33 @@
...
+* Fixed a case where re-rendering after rendering null didn't properly pass context
+* Fixed a case where re-rendering after rendering null didn't properly update style
+* Update `uglify` dependency to prevent a bug in IE8
+* Improved warnings
+
+### React with Add-Ons
+
+#### Bug Fixes
+
+* Update: Ensure it supports `hasOwnProperty` as an object key
```
Comment (spicyj) [inline CHANGELOG.md]: Maybe "Immutability Helpers" or "React.addons.update" for clarity?

File: npm-react/package.json [hunk 4]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "react",
   "description": "React is a JavaScript library for building user interfaces.",
-  "version": "0.13.1",
+  "version": "0.13.2-alpha",
```
Comment (spicyj) [inline npm-react/package.json]: Is this supposed to be -alpha?

**Comment by spicyj** [comment]
lgtm

File: CHANGELOG.md (hunk 1 above)
Comment (zpao) [inline CHANGELOG.md]: Ah true

File: npm-react/package.json (hunk 4 above)
Comment (zpao) [inline npm-react/package.json]: It is for my branch, the last commit will be the one that bumps to 0.13.2 and then that gets tagged.

**Comment by zpao** [comment]
Now with a minimal blog post. Let me know if you have any comments on it. I'll probably push this out in a little bit when I get to reliable internet.

**Comment by spicyj** [comment]
(y)

**Pull request merged**